### PR TITLE
chore: face auth adjustments for smaller devices

### PR DESCRIPTION
**Pull request by ice-hades**

### CODE REVIEW

File: src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx [hunk 1]
```diff
@@ -65,14 +66,13 @@ export const CameraFeed = React.forwardRef(
     ) : null;
   },
 );
-
 export const cameraStyles = StyleSheet.create({
   cameraContainer16to9: {
     width: windowWidth,
-    height: (windowWidth * 16) / 9,
+    height: Math.min(windowHeight - HEADER_HEIGHT, (windowWidth * 16) / 9),
```
Comment (ice-orion) [inline src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx]: don't u need to take notches into account here?

File: src/constants/styles.ts [hunk 2]
```diff
@@ -40,4 +40,4 @@ export const MIDDLE_BUTTON_HIT_SLOP = {
 export const windowWidth = Dimensions.get('window').width;
 export const windowHeight = Dimensions.get('window').height;
 
-export const smallHeightDevice = screenHeight < 680;
+export const smallHeightDevice = screenHeight < 750;
```
Comment (ice-hector) [inline src/constants/styles.ts]: We use it in many other places (around 20). Are you sure that on devices with heights between 680 and 750, UI will work as expected? Definitely not and should be carefully rechecked if you apply this

File: src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx (hunk 1 above)
Comment (ice-orion) [reply src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx]: And if i remember correctly on android if the camera ratio that you pass as a prop and the ratio of the camera component are different, then there'll be a deformation of the camera preview. Or i miss something?

Comment (ice-orion) [reply src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx]: May be just make the camera component of the width / height like it was before but with the absolute style and the FaceAuthOverlay make of `flex:1` - then that overlay will take all available height and the buttons will be always on the screen?

File: src/constants/styles.ts (hunk 2 above)
Comment (ice-hades) [reply src/constants/styles.ts]: On android Pixel 1 emulator with 5 inches display the height is "screenHeight": 731.4285714285714
5 inches is small device

Comment (ice-hector) [reply src/constants/styles.ts]: One device specs mean nothing for such a change. Please find the devices with 690-710 screen height and check all places (ideally make the screenshots and provide them) on how that looks like for all those 20 cases where we use `smallHeightDevice` before and after your fix

File: src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx (hunk 1 above)
Comment (ice-hades) [reply src/screens/FaceRecognitionFlow/components/CameraFeed/CameraFeed.tsx]: fixed

File: src/constants/styles.ts (hunk 2 above)
Comment (ice-hades) [reply src/constants/styles.ts]: fixed

**Review by ice-hector — DISMISSED** [review]

**Review by ice-hector — DISMISSED** [review]

**Review by ice-atlas — DISMISSED** [review]

**Review by ice-orion — DISMISSED** [review]

**Review by ice-atlas — DISMISSED** [review]

**Review by ice-orion — DISMISSED** [review]

**Review by ice-hector — APPROVED** [review]

**Review by ice-atlas — APPROVED** [review]

**Review by ice-orion — APPROVED** [review]

**Pull request merged**